content/kubermatic/master/startio/guides/wizard/_index.en.md

```diff
@@ -1,44 +1,47 @@
 +++
 title = "Use the Wizard to Configure KKP"
 weight = 10
+enableToc = true
 +++
 
 Start by visiting [https://start.kubermatic.io/](https://start.kubermatic.io/).
 
 Introduction page will welcome you.
 
-![Welcome Page](welcome.png "Welcome Page" )
-
 Start by clicking on the **Generate** button.
 
-## 1. Provider selection
-![Step 1](1.png "Step 1")
+![Welcome Page](welcome.png?width=700px&classes=shadow,border "Welcome Page" )
 
+## 1. Provider selection
 At this step, you will select a cloud provider where the KKP will be deployed.
 
-## 2. Kubernetes Cluster configuration
-![Step 2](2.png "Step 2")
+![Step 1](1.png?width=700px&classes=shadow,border "Step 1")
 
+## 2. Kubernetes Cluster configuration
 At this step, you are providing details of your Kubernetes master cluster.
 
-You can generate a _Cluster name_, provide a Kubernetes _Master Version_ (these are hard-coded to latest 1.20.x and 1.21.x versions now),
-select the preferred _Container Runtime_.
+You can generate a _Cluster name_, provide a Kubernetes _Master Cluster Version_.
 
 For the AWS setup, you can select an _AWS region_ where the cluster will be provisioned and also the _AWS Worker Type_
 (see the [Instance Types](https://aws.amazon.com/ec2/instance-types/), we recommend at least _t3.xlarge_ for the initial setup).
```

Comment:
How many nodes do we configure? 3?

Reply:
yes, 3 by default... I will also check whether it's regional or one-region setup... and I would also consider deploying autoscaler addon OOB, wdyt?